src/main/java/ui/components/pickers/PickerMilestone.java

```diff
@@ -153,20 +154,25 @@ private void setRemovedInUI(Label milestone) {
                 .findFirst();
     }
 
-    public void setSelected(boolean isSelected) {
-        this.isSelected = isSelected;
+    private void setNotMatchingInUI(Label milestone) {
+        milestone.setStyle(milestone.getStyle() + " -fx-opacity: 60%;");
     }
 
-    public boolean isSelected() {
-        return this.isSelected;
+    public PickerMilestone setMatching(boolean isMatching) {
```

Comment:
Does it need the return value here? Seems the return value is not used anywhere.